tests/queries/0_stateless/03594_is_valid_ascii_errors.reference

```diff
@@ -0,0 +1,4 @@
+Code: 43. DB::Exception: Cannot apply function isValidASCII to column Array(UInt8), expected String or FixedString: In scope SELECT isValidASCII([1, 2, 3]). (ILLEGAL_TYPE_OF_ARGUMENT)
+Code: 43. DB::Exception: Cannot apply function isValidASCII to column UUID, expected String or FixedString: In scope SELECT isValidASCII(toUUID('00000000-0000-0000-0000-000000000000')). (ILLEGAL_TYPE_OF_ARGUMENT)
+Code: 43. DB::Exception: Cannot apply function isValidASCII to column IPv6, expected String or FixedString: In scope SELECT isValidASCII(toIPv6('::1')). (ILLEGAL_TYPE_OF_ARGUMENT)
+Code: 43. DB::Exception: Cannot apply function isValidASCII to column IPv4, expected String or FixedString: In scope SELECT isValidASCII(toIPv4('127.0.0.1')). (ILLEGAL_TYPE_OF_ARGUMENT) 
```

Comment:
The `serverError` annotation checks error code while testing, but do not print error message, that's why reference should be empty. If we needed to check error messages we could use `.sh` test , but here it's required.

```suggestion
```